CHANGELOG.md

```diff
@@ -22,6 +22,8 @@
 * Fixed bug when using `Sequence.iter_kmers` on empty `Sequence` object. Previously this raised a `ValueError`, now it returns
 an empty generator.
 * Fixed minor bug where adding sequences to an empty `TabularMSA` with MSA-wide `positional_metadata` would result in a `TabularMSA` object in an inconsistent state. This could happen using `TabularMSA.append` or `TabularMSA.extend`. This bug only affects a `TabularMSA` object *without* sequences that has MSA-wide `positional_metadata` (for example, `TabularMSA([], positional_metadata={'column': []})`).
+* `TreeNode.distance` now handles the situation in which `self` or `other` are ancestors. ([#807](https://github.com/biocore/scikit-bio/issues/807))
```

Comment:
For both of these bugs, can you please describe what the previous behavior was? We're making an effort to note that for bug fixes in the changelog.